### DISCUSSION TITLE
caching with unstable_cache is not working

### DISCUSSION THREAD

**Discussion by nassmim** [vercel/next.js#73192] [Help]
### Summary

After several attempts and thinking I had a mistake in my code, I tried the exemple from official next documentation about unstable_cache.

```
const getCachedUser = unstable_cache(
  async (id) => getEmployerByAuthID(id),
  ["my-app-user"],
);

export default async function MyProfileInfo() {
  const { authUser, userData } = await getAuthUser({
    redirectionOnError: pages.login,
  });

  let employer;
  try {
    employer = await getCachedUser(authUser.id);
  } catch (err) {
    console.log(err);
  }
}
```

Inside getEmployerByAuthID, I have a console log to see if the function has been called or not. And it's always executed whereas I was expecting the result to come from the cache.

Is unstable_cache still working? I don't want to switch to nextjs15 right now. 
Or is there some configuration to make it work?

**Comment by lsli8888** [comment]
I'm encountering the same issue, too, in nextjs 14 app router.  But my function is even simpler as it doesn't have any parameters.  So this takes changes in parameter values/arguments out of the equation.

**Reply by nassmim** [reply]
@lsli8888 should we put this as an issue rather than a discussion? I even tried out the simplest exemple, just using the exact same exemple from nextjs documentation. Even this one does not work as expected. 
I suspect we might be missing some additional configuration to set, but nextjs does not mention it in its docs

**Reply by lsli8888** [reply]
I feel like this is an issue, but we may get help if it's also a discussion.  I think the admins may not like if it's posted in two places, but this may be the only way to get help!

**Reply by nassmim** [reply]
I created an issue here:
https://github.com/vercel/next.js/issues/73365#issue-2706250555
